.github/workflows/my-ci-pipeline.yml

```diff
@@ -23,28 +23,21 @@ jobs:
     name: Build and image
     runs-on: ubuntu-latest
     needs: build
-    if: github.ref_name == 'module8_updated'
+    if: github.ref_name == 'module-9'
     steps:
     - run: echo "Publishing!"
     - uses: actions/checkout@v2
     - name: docker build
-      run: docker build -t zorion75/todo-app-module8 .
+      run: docker build -t zorion75/todo-app .
     - name: login
       run: echo ${{secrets.DOCKER_HUB_TOKEN}} | docker login --username zorion75 --password-stdin
     - name: push image to docker hub
-      run: docker push zorion75/todo-app-module8
+      run: docker push zorion75/todo-app
 
-  heroku:
-    name: deplay to heroku
+  webhook:
+    name: Azure
     runs-on: ubuntu-latest
     needs: dockerhub
     steps:
-    - uses: actions/checkout@v2
-    - uses: akhileshns/heroku-deploy@v3.12.12
-      with:
-        heroku_api_key: ${{secrets.HEROKU_API}}
-        heroku_app_name: "todo-app-kat-module8"
-        heroku_email: "katarzyna.polonkiewicz@kpmg.co.uk"
-        usedocker: true
-
-          #hello
\ No newline at end of file
+      - name: azure
+        run: curl -dH -X POST ${{secrets.WEBHOOK_URL}}
```

Comment:
One nice addition here can be adding the [--fail](https://curl.se/docs/manpage.html#-f) flag, as otherwise curl reports success as long as a successful connection was made, even if the response reports a failure (e.g. a status code > 400) which can lead to confusing pipeline results